src/features/userSlice/state.js

```diff
@@ -10,6 +10,11 @@ const initialState = {
   fromUserHistory: historyState.none,
   userHistoryList: [],
   selectedHistory: null,
+  alertState: {
+    open: false,
+    severity: 'error',
```

Comment:
I think severity implies an impact, later on you set severity to "success" which doesn't seem like a level of severity.  Unless this is a standard I'm not aware of maybe we should use something like "type" or "condition"

Reply:
Yes this is a default 'type' for the alert component, but I can change the naming for this to make it clearer.